doc/source/data/dataset-tensor-support.rst

```diff
@@ -55,7 +55,7 @@ If you already have a Parquet dataset with columns containing serialized tensors
 
     # Write the dataset to Parquet. The tensor column will be written as an
     # array of opaque byte blobs.
-    ds = ray.data.from_pandas([ray.put(df)])
+    ds = ray.data.from_pandas([df])
```

Comment:
How about we take `*args` for from_pandas()? I feel the common case is passing just a single df if not passing distributed refs, so accepting a list is a bit weird.

Reply:
@ericl Agreed, I'm primarily trying to not step on [this PR's](https://github.com/ray-project/ray/pull/18205) toes!